source/module_io/input.h

```diff
@@ -603,6 +603,34 @@ class Input
     double qo_thr = 1e-6;
     std::vector<std::string> qo_strategy = {};
     std::vector<double> qo_screening_coeff = {};
+    //==========================================================
```

Comment:
please add these parameters in `input_test_para.cpp`

Reply:
ok